src/media_tools/av_parsers.c

```diff
@@ -6516,7 +6605,7 @@ static void avc_hevc_vvc_rewrite_vui(GF_VUIInfo *vui_info, GF_BitStream *orig, G
 		}
 	}
 
-	if (is_vvc && vui_chroma_loc_info_present_flag) {
+	if (codec == GF_CODECID_VVC && vui_chroma_loc_info_present_flag) {
```

Comment:
could you put parenthesis around this check ? Apart from that looks good to me

Reply:
done